Nos/Service/FileStorage/FileStorageAPIClient.swift

```diff
@@ -54,18 +65,19 @@ class NostrBuildAPIClient: FileStorageAPIClient {
             let apiURL = URL(string: apiURLString) else {
             throw FileStorageAPIClientError.invalidURLRequest
         }
-
-        let (request, data) = try uploadRequest(fileAt: fileURL, apiURL: apiURL)
-        let (responseData, _) = try await URLSession.shared.upload(for: request, from: data)
-
+        
+        return apiURL
+    }
+    
+    /// The URL of the uploaded asset parsed from the API's response.
+    private func assetURL(from responseData: Data) throws -> URL {
```

Comment:
Thanks! I meant to explain it a little bit. Initially I tried using `PhotosPicker` because it was about 5 lines of code to get full functionality (minus camera access), and with that it's easier to upload just a `Data`. So I had a function for that as well as part of the protocol for `FileStorageAPIClient`. But then transitioning to the existing image picker method, I didn't need that anymore, so I removed it. Would be easy to add back if we ever need it though.